mmv1/third_party/terraform/fwtransport/framework_config.go.erb

```diff
@@ -31,25 +31,26 @@ import (
 )
 
 type FrameworkProviderConfig struct {
-	BillingProject             types.String
-	Client                     *http.Client
-	Context                    context.Context
-	gRPCLoggingOptions         []option.ClientOption
-	PollInterval               time.Duration
-	Project                    types.String
-	Region                     types.String
-	Zone                       types.String
-	RequestBatcherIam          *transport_tpg.RequestBatcher
-	RequestBatcherServiceUsage *transport_tpg.RequestBatcher
-	Scopes                     types.List
-	TokenSource                oauth2.TokenSource
-	UserAgent                  string
-	UserProjectOverride        types.Bool
-
-	// paths for client setup
-	<% products.each do |product| -%>
-	<%= product[:definitions].name -%>BasePath string
-	<% end -%>
+    BillingProject types.String
```

Comment:
Can you fix the formatting on these lines?  They were previously tabbed, consistent with the rest of the file, but seem to be spaces now.